Week-3/day6/tictactoe/styles.css

```diff
@@ -0,0 +1,30 @@
+body {
+  font-family: Roboto, Helvetica, sans-serif;
+  text-align: center;
+}
+
+.x {
+  color: red;
+}
+
+.o {
+  color: blue;
+}
+
+#board {
+  font: 85px arial, sans-serif;
+  margin: 3px;
+  
+  /* Your solution goes here */      
+  width: 300px;
+  aspect-ratio: 1;
+  margin: auto;
+  display: grid;
+  grid-template-columns: repeat(3, 33%);
```

Comment:
The grid-template-columns property could use 1fr instead of percentages for a more flexible layout that doesn't depend on a fixed percentage.